src/fastga/models/aerodynamics/external/openvsp/openvsp.py

```diff
@@ -1258,8 +956,234 @@ def read_results(result_file_path):
+        s_ref_wing = float(inputs["data:geometry:wing:area"])
+        s_ref_htp = float(inputs["data:geometry:horizontal_tail:area"])
+        area_ratio = s_ref_htp / s_ref_wing
+        sweep25_wing = float(inputs["data:geometry:wing:sweep_25"])
+        taper_ratio_wing = float(inputs["data:geometry:wing:taper_ratio"])
+        aspect_ratio_wing = float(inputs["data:geometry:wing:aspect_ratio"])
+        sweep25_htp = float(inputs["data:geometry:horizontal_tail:sweep_25"])
+        aspect_ratio_htp = float(inputs["data:geometry:horizontal_tail:aspect_ratio"])
+        taper_ratio_htp = float(inputs["data:geometry:horizontal_tail:taper_ratio"])
+        dihedral_angle = float(inputs["data:geometry:wing:dihedral"])
+        twist_angle = float(inputs["data:geometry:wing:twist"])
+        geometry_set = np.around(
+            np.array(
+                [
+                    sweep25_wing,
+                    taper_ratio_wing,
+                    aspect_ratio_wing,
+                    dihedral_angle,
+                    twist_angle,
+                    sweep25_htp,
+                    taper_ratio_htp,
+                    aspect_ratio_htp,
+                    mach,
+                    area_ratio,
+                ]
+            ),
+            decimals=6,
+        )
+
+        return s_ref_wing, area_ratio, geometry_set
+
+    @staticmethod
+    def post_process_wing(inputs, wing_0, wing_aoa, s_ref_wing, aoa_angle):
+        """
+        Computes wing data not produced by OpenVSP based on available ones
+
+        :param inputs: inputs in the OpenMDAO format
+        :param wing_0: wing aerodynamic coefficient with 0 angle of attack
+        :param wing_aoa: wing aerodynamic coefficient with input angle of attack
+        :param s_ref_wing: wing reference surface area
+        :param aoa_angle: input angle of attack
+
+        :return: lift coefficient at 0 aoa, lift coefficient at input aoa, lift slope coefficient,
+        pitching moment coefficient, span vector, lift coefficient vector, chord vector,
+        lift induced drag coefficient
+        """
+        width_max = inputs["data:geometry:fuselage:maximum_width"]
+        span_wing = inputs["data:geometry:wing:span"]
+        k_fus = 1 + 0.025 * width_max / span_wing - 0.025 * (width_max / span_wing) ** 2
+        cl_0_wing = float(wing_0["cl"] * k_fus)
+        cl_x_wing = float(wing_aoa["cl"] * k_fus)
+        cm_0_wing = float(wing_0["cm"] * k_fus)
+        cl_alpha_wing = (cl_x_wing - cl_0_wing) / (aoa_angle * np.pi / 180)
+        y_vector_wing = wing_aoa["y_vector"]
+        cl_vector_wing = (np.array(wing_aoa["cl_vector"]) * k_fus).tolist()
+        chord_vector_wing = wing_aoa["chord_vector"]
+        k_fus = 1 - 2 * (width_max / span_wing) ** 2  # Fuselage correction
+        # Full aircraft correction: Wing lift is 105% of total lift, so: CDi = (CL*1.05)^2/(
+        # piAe) -> e' = e/1.05^2
+        coeff_e = float(wing_aoa["coeff_e"] * k_fus / 1.05 ** 2)
```

Comment:
I meant this particular correction on the Oswald coefficient. Line 1178 of vlm.py file, the 1.05 factor is not present. I wonder why this difference compare to openvsp.

Reply:
I see now, sorry for my previous message. As for the reason for that coefficient, it seems to have been here from the start. I can't really tell why it has been added.